static/w3id-security-v1.json

```diff
@@ -0,0 +1,49 @@
+{
+    "@context": {
+      "id": "@id",
```

Comment:
Code standards: Inconsistent indentation, please choose 2 or 4-spaces indentation for the whole file.